static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -75,7 +75,11 @@ define([
                 },
                 shouldSendCallback: function (data) {
                     var isDev = config.page.isDev;
-                    if (isDev) {
+
+                    var isIgnored = typeof(data.tags.action) !== 'undefined' &&
+                        config.page.ignoredSentryActions.indexOf(data.tags.action) != -1;
```

Comment:
@jfsoul 

right on the != needing to be !==

As for just relying on the falsyness of undefined; it depends what we want to do. It is possible for the action field not to be set, and I've always considered that since JS has had some historically (currently?) weird rules for equality handling, that being explicit about the intent of the code trumps the extra characters, hence me putting in explicitly !== undefined since that carries with it the hint to the programmer that the value can actually be undefined as an expected case.